### ISSUE TITLE
PurgeLinesAndUnload, Ignore commented out purge lines in startup gcode

### ISSUE THREAD

**Issue by PartySausage** [GregValiant/GV-PostProcessors#1]
Hi Greg. I installed this rather good script so that I can easily move/omit my purge lines when printing large models. 
In the short term; Rather than delete my current purge lines from my start-up gcode I just commented them out, however the search string in the script still identifies these as active purge lines & displays the warning.

Can I suggest changing the search string to omit commented out purge lines from the search? 
I tried the following which seems to work **"[^;*]+G1 X(\d.*) Y(\d.*) E(\d.*)"** What do you think?

**Comment by GregValiant** [comment]
Nice catch.  This is what I'm looking for - a nice concise critique.

I've made the change and updated the file.  Instead of using a regular expression I've changed it to step through the startup and ignore commented lines.  That seems to work fine.
Some printer definitions have the "F" parameter in different locations in the line and that appears to have caused heartburn for the ReGex search.

From a newer Creality definition file:  G1 X0.1 Y200.0 Z0.3 **F1500.0** E15 from an older one:  G1 **F1500** X0.1 Y200.0 Z0.3 E15
The Elegoo definition has no "X" parameter:  G1 Y120.0 F600.0 E20
